python/private/python.bzl

```diff
@@ -653,7 +804,14 @@ error to run with root access instead.
         ),
         "is_default": attr.bool(
             mandatory = False,
-            doc = "Whether the toolchain is the default version",
+            doc = """\
+Whether the toolchain is the default version.
+
+:::{versionchanged} VERSION_NEXT_FEATURE
+This setting is ignored if the default version is set using the `defaults`
+tag class.
+:::
```

Comment:
Should we deprecate this in favour of the `defaults` tag_class?

cc: @rickeylev what do you think?